cremalab/crema-app-web
# Update create-react-app to v4 🎉

## Description

This PR updates create-react-app to v4 as well as fixes and updates everything to be more in line with what a fresh create-react-app looks like. Closes #397

With this update windows should now be supported fully and work in CI. Closes #365

I also took the time to add a little Crema flair. 😉

### 🔗 Reference

> _Relevant links (e.g. issue, design, etc.)_

- [Update template to use create-react-app@v4](https://github.com/cremalab/app-web/issues/397)

### 📋 Acceptance Criteria

> _Checked off by PR **Assignees**_

- [x] App Builds
- [x] Fast refresh works

### 🖼 Demo

![fast-refresh](https://user-images.githubusercontent.com/6101761/97755089-8f99cc00-1ac6-11eb-88e5-1b958131554a.gif)



### 👩‍🔬 Test Instructions

> _Provided by PR **Assignees**_

1. Pull down branch
1. `npm ci`
1. `npm start`
1. Ensure app works as expected
1. `npm run test:ci`
1. Ensure tests pass

## 🔎 Reviewer Checklist

> _Checked off by PR **Reviewers**_




---

> ### Roles & Responsibilities
>
> #### 👨‍💻 Assignee
>
> - Initiator of this PR (be sure to set in GitHub UI)
> - Addresses feedback and change requests
> - Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
> #### 👩‍💻 Reviewer
>
> - Invited to review PR by **Assignee** (via GitHub UI)
> - Is expected to complete a review and address followup
